### PR TITLE
openafs_client install-build: Use synchronize

### PR DESCRIPTION
**Pull request by CheyenneWills**
Replace rsync command with synchronize.

Note:
Not sure why, but could only get proper results by splitting the 
synchronize into two separate tasks.

### CODE REVIEW

**Comment by CheyenneWills** [comment]
There is a reference to delegate_to in the ansible 0.7 changelog (Sept 2012)

**Comment by CheyenneWills** [comment]
I noted that at one point I was having some odd behavior when there was just a single sync.  /lib/modules wasn't getting updated correctly.  I will continue to investigate to see if the current set of options "fixed" that.

Odd behavior: /lib/modules was cleaned out except for what was being copied in one case and in another the data wasn't copied.

**Comment by meffie** [comment]
Sorry for the delay here. Did we still need this pull request to build on centos?

**Comment by CheyenneWills** [comment]
I think so.  I'm still using synchronize instead of rsync.  Just had to make it two separate tasks for some reason..

File: roles/openafs_client/tasks/redhat/install-build.yaml [hunk 1]
```diff
@@ -84,10 +84,24 @@
     path: /tmp/openafs-client-build/usr/afs
     state: absent
 
-- name: Install files
-  command: rsync -av * /
-  args:
-    chdir: /tmp/openafs-client-build
+# Needed to split the install into separate tasks
+# in order to get correct result,
+# /lib/modules was not updating correctly
```
Comment (meffie) [inline roles/openafs_client/tasks/redhat/install-build.yaml:89]: I think i'd prefer to have this commentary in the commit message.

Comment (meffie) [reply roles/openafs_client/tasks/redhat/install-build.yaml:89]: Ah, i see that commentary is in the commit message already. I'll merge as is, and we can clean this up later.

**Pull request merged**